ui/src/components/GraphComponents/GraphGradient.tsx

```diff
@@ -0,0 +1,23 @@
+export const GraphGradient: React.FC = () => {
+  return (
+    <svg width="20" height="20">
+      {/* This makes a gradient background available to the custom edges */}
+      <defs>
+        <linearGradient id="gradient" x1="0%" y1="0%" x2="100%" y2="0%">
+          <stop offset="0%" stopColor="#4abaeb" />
+          <stop offset="100%" stopColor="#5dc4b6" />
```

Comment:
This is pretty nitpicky, but I notice hard edges where the gradient starts/ends, it makes these line artifacts appear - might be smoother to transition back to the start color? Might also make sense to double the pattern and rect width below if you decide to take this change.
```suggestion
          <stop offset="0%" stopColor="#4abaeb" />
          <stop offset="50%" stopColor="#5dc4b6" />
          <stop offset="100%" stopColor="#4abaeb" />
```

Reply:
Good idea